tests/image/classification/test_data.py

```diff
@@ -611,3 +616,31 @@ def mixup(batch, alpha=1.0):
 
     batch = next(iter(img_data.train_dataloader()))
     assert "lam" in batch["metadata"][0]
+
+
+def test_data_source_collection(single_target_csv, tmpdir):
+
+    dm = ImageClassificationDataModule(
+        loader=ImageClassificationLoader.from_csv(
+            "image",
+            "target",
+            train_file=single_target_csv,
+            predict_file=single_target_csv,
+        )
+    )
```

Comment:
Since ImageClassificationDataModule already has the loader cls defined as a static property, is there a way to do this where you don't have to specify to the API to use the data loader already defined in the data module it's creating?